splinepy/bezier.py

```diff
@@ -4,6 +4,57 @@
 class BezierBase(spline.Spline):
     """Bezier Base. Contain extra operations that's only
     available for bezier families.
+
+    Bezier splines are a special type of spline where the basis functions have
+    a global support. The basis functions are given by the so-called
+    *Berstein polynomials*
+
+    .. math::
+            N_{i;p}(u) = \\binom{p}{i} u^i (1-u)^{p-i}
+
+    A Bezier spline itself can be seen as a map from the parametric domain
+    :math:`\\[0,1\\]^{N_{param}}` to the physical domain
+    :math:`\\mathbb{R}^{N_{phys}}`. We can distinguish between different types
+    of splines depending on the dimension of the parametric space.
+
+    #. A spline of degree :math:`p` with control points
+    :math:`P_i\\in\\mathbb{R}^{N_{phys}` and a one-dimensional parameter space
+    corresponds to a line embedded into the physical space:
+
+    .. math::
+            C(u) = \\sum_{i=0}^{l} N_{i;p}(u) P_i
+
+    #. A spline of degrees :math:`p,q` with control points
+    :math:`P_{i,j}\\in\\mathbb{R}^{N_{phys}` and a two-dimensional parameter
+    space corresponds to a surface, embedded into the physical space:
+
+    .. math::
+            C(u,v) = \\sum_{i=0}^{l} \\sum_{j=0}^{m} N_{i;p}(u) N_{j;q}(v)
+                P_{i,j}
+
+    Due to the tensor-product nature of the Bezier basis functions, this is
+    often rewritten as in terms of multi-variate basis functions
+
+    .. math::
+            \\tilde{N}_{i,j;p,q}(u,v) \\coloneqq N_{i;p}(u) N_{j;q}(v)
+
+    #. A spline of degrees :math:`p,q,r` with control points
+    :math:`P_{i,j,k}\\in\\mathbb{R}^{N_{phys}` and a three-dimensional
+    parameter space corresponds to a surface, embedded into the physical space:
```

Comment:
```suggestion
    parameter space corresponds to a volume, embedded into the physical space:
```
I think we got the idea with surfaces already, but this can stay. Please also mention, that parametric and physical domain can be different in a general case and that we support up to 10 dimensions for both the parametric and physical domain seperately

Reply:
In the documentation, they are not assumed to be equal, that's why different symbols for the parametric and physical domains are used. But I'm now going to add additional information to the documentation and store them in the `Spline` class.